app/src/main/java/org/thoughtcrime/securesms/database/helpers/SQLCipherOpenHelper.java

```diff
@@ -125,9 +126,9 @@ public void postKey(SQLiteConnection connection) {
           // if not vacuumed in a while, perform that operation
           long currentTime = System.currentTimeMillis();
           // 7 days
-          if (currentTime - TextSecurePreferences.getLastVacuumTime(context) > 604_800_000) {
+          if (currentTime - MessagingModuleConfiguration.getShared().getPrefs().getLastVacuumTime() > 604_800_000) {
```

Comment:
~~Out of curiosity, why do you reckon it might be 604,800,000... that's oddly specific, isn't it?~~

Ah, it's 1 week in milliseconds. Can we use something like `1.weeks.inWholeMilliseconds` instead?

Reply:
totally, that's `604_800_000 x` thinking.

Although... I could add a courtesy `Pref` with an `Converter` to get/set `Duration`s directly... 🤔